packages/@aws-cdk/aws-stepfunctions-tasks/lib/emrcontainers/start-job-run.ts

```diff
@@ -0,0 +1,681 @@
+import * as path from 'path';
+import * as iam from '@aws-cdk/aws-iam';
+import * as lambda from '@aws-cdk/aws-lambda';
+import * as logs from '@aws-cdk/aws-logs';
+import * as s3 from '@aws-cdk/aws-s3';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import * as cdk from '@aws-cdk/core';
+import * as cr from '@aws-cdk/custom-resources';
+import * as awscli from '@aws-cdk/lambda-layer-awscli';
+import { Construct } from 'constructs';
+import { integrationResourceArn, validatePatternSupported } from '../private/task-utils';
+
+/**
+ * The props for a EMR Containers StartJobRun Task.
+ */
+export interface EmrContainersStartJobRunProps extends sfn.TaskStateBaseProps {
+
+  /**
+   * The ID of the virtual cluster where the job will be run
+   */
+  readonly virtualCluster: VirtualClusterInput;
+
+  /**
+   * The name of the job run.
+   *
+   * @default - No job run name
+   */
+  readonly jobName?: string;
+
+  /**
+   * The execution role for the job run.
+   *
+   * If `virtualClusterId` is from a JSON input path, an execution role must be provided.
+   * If an execution role is provided, follow the documentation to update the role trust policy.
+   * @see https://docs.aws.amazon.com/emr/latest/EMR-on-EKS-DevelopmentGuide/setting-up-trust-policy.html
+   *
+   * @default - Automatically generated only when the provided `virtualClusterId` is not an encoded JSON path
+   */
+  readonly executionRole?: iam.IRole;
+
+  /**
+   * The Amazon EMR release version to use for the job run.
+   */
+  readonly releaseLabel: ReleaseLabel;
+
+  /**
+   * The configurations for the application running in the job run.
+   *
+   * Maximum of 100 items
+   *
+   * @see https://docs.aws.amazon.com/emr-on-eks/latest/APIReference/API_Configuration.html
+   *
+   * @default - No application config
+   */
+  readonly applicationConfig?: ApplicationConfiguration[];
+
+  /**
+   * The job driver for the job run.
+   *
+   * @see https://docs.aws.amazon.com/emr-on-eks/latest/APIReference/API_JobDriver.html
+   */
+  readonly jobDriver: JobDriver;
+
+  /**
+   * Configuration for monitoring the job run
+   *
+   * @see https://docs.aws.amazon.com/emr-on-eks/latest/APIReference/API_MonitoringConfiguration.html
+   *
+   * @default - logging enabled and resources automatically generated if `monitoring.logging` is set to `true`
```

Comment:
The monitoring configuration is based on monitoring.logging variable. There are three use cases:
1) The user says they do not want monitoring at all 
2) The user wants auto generated s3 bucket and logging configurations 
3) The user provides their own s3 bucket and logging configurations 

In this case the default should actually be false, since monitoring.logging by default is set to false